pkg/envoy/route/config.go

```diff
@@ -29,27 +29,21 @@ const (
 )
 
 const (
-	//InboundRouteConfigName is the name of the route config that the envoy will identify
+	// InboundRouteConfigName is the name of the route config that the envoy will identify
 	InboundRouteConfigName = "RDS_Inbound"
 
-	//OutboundRouteConfigName is the name of the route config that the envoy will identify
+	// OutboundRouteConfigName is the name of the route config that the envoy will identify
 	OutboundRouteConfigName = "RDS_Outbound"
 
-	// maxRegexProgramSize is the max supported regex complexity
-	maxRegexProgramSize = 1024
+	// InboundVirtualHost is the name of the virtual host on the inbound route configuration
+	InboundVirtualHost = "inbound_virtualHost"
 
-	//InboundVirtualHost is the name of the virtual host on the inbound route configuration
-	inboundVirtualHost = "inbound_virtualHost"
-
-	//OutboundVirtualHost is the name of the virtual host on the outbound route configuration
-	outboundVirtualHost = "outbound_virtualHost"
+	// OutboundVirtualHost is the name of the virtual host on the outbound route configuration
+	OutboundVirtualHost = "outbound_virtualHost"
```

Comment:
Why are these 2 constants being exported with this change? The first letter should be lower case to avoid exporting these.

Reply:
They shouldn't be. I changed them back and updated the comments instead. thanks.